src/frontend/src/views/Data/style.ts

```diff
@@ -8,3 +10,34 @@ export const Container = styled.div`
   flex-flow: column wrap;
   align-content: flex-start;
 `;
+
+export const LineageCell = styled.div``;
+
+export const LineageRowContent = styled(RowContent)`
+  &:hover {
+    ${LineageCell} {
```

Comment:
This is where we're using [Emotion Component as a selector](https://emotion.sh/docs/babel#components-as-selectors)